src/osemgrep/cli_scan/Scan_subcommand.ml

```diff
@@ -27,13 +27,22 @@ open Common
 
 module Out = Semgrep_output_v1_t
 module RP = Report
+module SS = Set.Make (String)
 
 (*****************************************************************************)
 (* To run the pro engine, including multistep rules *)
 (*****************************************************************************)
 
+type diff_config = { diff_targets : Fpath.t list; diff_depth : int option }
```

Comment:
Maybe add a comment like
(* This is used to filter matches from files that are not in the commit (see the --baseline-commit option). 
  * diff_targets contain the list of all files involved in a commit; diff_depth is ???
 *)

Reply:
Done. Added the comment to .mli file.